bentoml/_internal/frameworks/pytorch.py

```diff
@@ -86,13 +86,27 @@ def load(
                 f"Model {tag} was saved with module {bentoml_model.info.module}, failed loading with {MODULE_NAME}."
             )
     weight_file = bentoml_model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
-    # TorchScript Models are saved as zip files
-    if zipfile.is_zipfile(weight_file):
+    model_format = bentoml_model.info.options.get("_model_format")
+    # backward compatibility
+    if not model_format:
+        if zipfile.is_zipfile(weight_file):
+            model_format = "torchscript:v1"
+        else:
+            model_format = "cloudpickle:v1"
+
+    if model_format == "torchscript:v1":
         model: "torch.jit.ScriptModule" = torch.jit.load(weight_file, map_location=device_id)  # type: ignore[reportPrivateImportUsage] # noqa: LN001
-        return model
-    else:
+    elif model_format == "cloudpickle:v1":
         with Path(weight_file).open("rb") as file:
-            return cloudpickle.load(file)
+            model: "torch.nn.Module" = cloudpickle.load(file).to(device_id)
+    elif model_format == "torch.save:v1":
+        with Path(weight_file).open("rb") as file:
+            model: "torch.nn.Module" = torch.load(file, map_location=device_id)
+    else:
+        raise BentoMLException(f"Unknown model format {model_format}")
+
+    model.eval()
```

Comment:
should we set the model to `eval()` mode here?

Reply:
or should we do it in `_PyTorchRunner._setup()`? I think we should do `eval()` for user somewhere in our codes?

Reply:
there will be use-case where users load back, they want to continue train, so i think we can put `eval` under runner instead of here

Reply:
moved `eval` to `_setup()`